images/common/utils.sh

```
@@ -229,28 +229,27 @@ function openvpn_config {
+openvpn_config_checksum() {
 	export OFILE=$(curl --silent --insecure \
 		${API_INTERNAL}/controller/vpn/checksum/$UUID/?key=$KEY)
 	export NFILE=$(cat checksum)
 }
 
-function openvpn_config_download {
-	curl --silent --retry 10 --retry-delay 5 --retry-max-time 300\
-		--insecure --output vpn.tar.gz \
+openvpn_config_download() {
+	curl --silent --retry 10 --retry-delay 5 --retry-max-time 300 --insecure --output vpn.tar.gz \
 		${API_INTERNAL}/controller/vpn/download-config/$UUID/?key=$KEY
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)